.travis.yml

```diff
@@ -35,19 +35,10 @@ notifications:
 
 # command to run tests
 script:
-  - pipenv run coverage run --source=imapfw,imapfw.py -p -m imapfw.edmp
-  - pipenv run coverage run --source=imapfw,imapfw.py -p -m imapfw.mmp.manager
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -h
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py noop
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -c multiprocessing unitTests
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -c threading unitTests
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all noop
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all testRascal
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all examine
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c multiprocessing syncAccounts -a AccountA
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c threading syncAccounts -a AccountA
-  - pipenv run coverage run --source=imapfw,imapfw.py -p ./imapfw.py -r ./tests/syncaccounts.1.rascal -d all -c threading syncAccounts -a AccountA -a AccountA -a AccountA
-  - pipenv run coverage combine .coverage*
+  - make test
```

Comment:
Is this "make test" equivalent compared to all the tests as script? What will happen if one test fails? Will the next one be run anyway?

Reply:
Yes, they are equivalent. The first non-zero exit status will fail the whole build.